COMETwebapp/Scene.cs

```diff
@@ -30,16 +30,59 @@ namespace COMETwebapp
     using System.Threading.Tasks;
     using System.Collections.Generic;
 
+    using CDP4Common.SiteDirectoryData;
+
     using COMETwebapp.Primitives;
 
     using Newtonsoft.Json;
+
     using Microsoft.AspNetCore.Components;
 
     /// <summary>
     /// Static class to access the resources of a Scene 
     /// </summary>
     public static class Scene
     {
+        /// <summary>
+        /// Shape Kind parameter short name
+        /// </summary>
+        public const string ShapeKindShortName = "kind";
+
+        /// <summary>
+        /// Orientation parameter short name
+        /// </summary>
+        public const string OrientationShortName = "orientation";
+
+        /// <summary>
+        /// Position parameter short name
+        /// </summary>
+        public const string PositionShortName = "coord";
+
+        /// <summary>
+        /// Width parameter short name
+        /// </summary>
+        public const string WidthShortName = "wid_diameter";
+
+        /// <summary>
+        /// Diameter parameter short name
+        /// </summary>
+        public const string DiameterShortName = WidthShortName;

```

Comment:
is this correct?

Reply:
Hmm it is correct. The diameter and width share the shortName but in terms of calling the field I thought that I would be more clear if the names are different.